libs/fixtures/tsconfig.json

```diff
@@ -68,7 +68,7 @@
     "skipLibCheck": true /* Skip type checking of declaration files. */,
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
   },
-  "include": ["src", "src/data/*.json"],
+  "include": ["src", "src/data/*.json", "src/data/**/*.json"],
```

Comment:
`"src/data/**/*.json"` includes `"src/data/*.json"`, so you can just replace the latter with the former.